dev-docs/source/UserSupport.rst

```diff
@@ -31,8 +31,8 @@ Bugs and Error Reports
 1.	Users can report bugs via the `Mantid Help Forum <https://forum.mantidproject.org/>`_ or the `Mantid Help Email <mantid-help@mantidproject.org>`_, or from collected **Error Reports**. Currently this is a quick first contact with the team, but doesn't give much detail about the usage or unexpected error.
 2.	The bug is verified and reproduced by the support team.
 3.	The impact and importance is assessed by the support team member by contacting the users, instrument scientists, developers or project manager as appropriate.
-4.	A Github issue to resolve the problem is created if appropriate and/or workaround tested if possible.
-5.	The user is contacted to give a link to the created issue and/or workaround solution.
-6.	When any issue is completed naming a user, that user is contacted to let them know it will be available in the nightly build and next release.
+4.	A GitHub issue to resolve the problem is created if appropriate and/or workaround tested if possible.
+5.	The user is contacted to give a link to the created issue and/or workaround solution, by the support team.
+6.	When any issue is completed naming a user, that user is contacted to let them know it will be available in the nightly build and next release. This responsibility lies with the gatekeeper that merges the fix, but may be delegated to the appropriate developer.
```

Comment:
I think this is wrong.

This should not be the job of the gatekeeper, rather the original issue reporter, or the developer implementing the fix. The gatekeeper is there to make sure the changes are technically sensible, not be Mantid's PR rep.

They will already have a line of communication to the relevant person and gatekeepers may not have enough information to contact the person in question.

I certainly have been ignoring the named people in PRs assuming the developer will contact them if it is particularly important or time critical.

Reply:
@DanNixon So, Daniel is going to make this a bit more explicit, but it is important not to ignore this completely as gatekeepers.  For the developer they may already be hard at work on another issue when this PR is merged, and could forget that they were supposed to inform someone.  The gatekeeper has that name in front of them when merging, so we just need to include a final message on the PR saying something like "@developer plesae let INSERT_NAME_HERE know this has now been merged."  That way they will get a specific email and reminder to inform the people waiting for this.

Reply:
The developer will get an email when the PR is merged anyway. Personally I think this is a pointless shift in responsibility.

Reply:
I have made the alteration to mention that commenting @developer is appropriate